### DISCUSSION TITLE
Is there a PDF version for the MDN contents?

### DISCUSSION THREAD

**Discussion by dqwyy** [mdn/mdn-community#458] [Platform] [answered]
Thanks for making such a good platform for learning HTML, CSS, JS, etc.

I want to know that do you provide a typeset PDF file for the MDN contents? You know it might be a little inconvenient to read on the website, for example you have to click to turn to next page frequently. If there is a PDF version, then I can print it and read smoothly.

Thanks.

**Comment by yin1999** [comment, marked as answer]
Unfortunately, MDN have not provided PDF version of contents. There are many embeded many live samples (e.g. [Examples in Inline-level content](https://developer.mozilla.org/en-US/docs/Glossary/Inline-level_content#examples)) and interactive examples (e.g. ["Try it" in `getter`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get#try_it)) in MDN contents. And it would be hard to represent these examples in PDF.

But MDN has provided typesetting optimization for printing. If you really need the PDF version, try the print feature of the browser and save the web pages as PDF files :)

**Reply by dqwyy** [reply]
oh ok i see, thank you.
